### PR TITLE
refactor: rename x/collection events

### PR DESCRIPTION
**Pull request by 0Tech**
## Description
The x/collection events now use the past tense.
Also renamed `EventAbandon` into `EventRenounced` since the latter is already in use on other implementations.

## Motivation and context

To unify the event nomenclature of x/collection.

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [x] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

**Review by zemyblue — COMMENTED** [review]
Please check swagger file

**Review by zemyblue — APPROVED** [review]

File: x/collection/event.go [hunk 1]
```diff
@@ -499,7 +499,7 @@ func NewEventTransferNFTFrom(event EventSent) sdk.Events {
 }
 
 // Deprecated: use EventGrant.
```
Comment (dudong2) [inline x/collection/event.go:501]: EventGranted?

File: x/collection/event.go [hunk 2]
```diff
@@ -519,7 +519,7 @@ func NewEventGrantPermToken(event EventGrant) sdk.Event {
...
```
Comment (dudong2) [inline x/collection/event.go:521]: ditto

File: x/collection/event.go [hunk 3]
```diff
@@ -538,7 +538,7 @@ func NewEventGrantPermTokenHead(event EventGrant) sdk.Event {
...
```
Comment (dudong2) [inline x/collection/event.go:540]: ditto

File: x/collection/event.go [hunk 4]
```diff
@@ -553,7 +553,7 @@ func NewEventGrantPermTokenBody(event EventGrant) sdk.Event {
 }
 
 // Deprecated: use EventAbandon.
```
Comment (dudong2) [inline x/collection/event.go:555]: EventRenounced?

**Review by dudong2 — APPROVED** [review]

**Pull request merged**